projects/ng2-charts/src/lib/base-chart.directive.ts

```diff
@@ -409,7 +407,7 @@ export class BaseChartDirective implements OnChanges, OnInit, OnDestroy, DoCheck
 
     const mergedOptions = this.smartMerge(options, this.themeService.getColorschemesOptions());
 
-    const chartConfig: ChartConfiguration = {
+    return {
```

Comment:
I did this for type safety, but ok :)

Reply:
It's implicit in the function return type:
`getChartConfiguration(): ChartConfiguration`